src/text/skeleton.scss

```diff
@@ -3,21 +3,26 @@
 @import '../style/index';
 
 @mixin iui-text-skeleton {
-  display: inline-block;
   user-select: none;
   color: transparent;
   border-radius: $iui-border-radius;
+  cursor: wait;
 
   @include themed {
     background: linear-gradient(
-      270deg,
-      t(iui-color-background-3),
-      t(iui-color-background-3),
-      t(iui-color-background-2),
-      t(iui-color-background-3),
-      t(iui-color-background-3)
+      292deg,
+      t(iui-text-color-muted),
+      t(iui-text-color-muted),
+      t(iui-text-color-placeholder),
+      t(iui-text-color-muted),
+      t(iui-text-color-muted)
     );
-    background-size: 200% 100%;
+  }
+  background-size: 200% 100%;
+
+  @media (forced-colors: active) {
+    forced-color-adjust: none;
```

Comment:
So when forced colors are turned on, should this marker in middle be moving a little? Looks weird..
![typography](https://user-images.githubusercontent.com/81580355/163543072-68f1c138-7efd-494d-8166-eb4496f2317b.gif)

Reply:
Check here 😀
https://adrianroselli.com/2020/11/more-accessible-skeletons.html#Update04